runtime/runtime.c

```diff
@@ -253,9 +253,41 @@ void* allocate_n_bytes_ptr(u32 n) {
 WEAK void populate_globals() {}
 
 // Code that actually runs the wasm code
-IMPORT i32 wasmf_main(i32 a, i32 b);
+IMPORT void wasmf__start(void);
```

Comment:
This diff makes us super wasi specific--prefer if you push this decision into the platform specific file